### PR TITLE
Merge upstream v0.2.6

### PR DESCRIPTION
**Pull request by meyerj**
Merge upstream version [v0.2.6](v0.2.6) and

- remove the `--mount` extension (originally https://github.com/osrf/rocker/pull/46) in favor of `--volume` added in https://github.com/osrf/rocker/commit/dc046c7d0965892d5dbb8156aefa8306c74fd4fb, and make `--mount` an alias for `--volume` for backwards compatibility (c81c2ffa59223519bda810611ac981e97019b61f)
- do not tag `detect_os` images, see also https://github.com/Intermodalics/rocker/pull/6#issuecomment-941629368 and https://github.com/osrf/rocker/pull/133#issuecomment-941622117  (b9637f7fb47f8596c8b5bdd95036dc21e1609841)

### CODE REVIEW

**Comment by spd-intermodalics** [comment]
It seems working for me as well and it tracks upstream mostly, so LGTM

**Review by spd-intermodalics — APPROVED** [review]
LGTM

**Comment by spd-intermodalics** [comment]
Although,

```
$ sudo dpkg --configure python3-rocker
Setting up python3-rocker (1:0.2.6-1intermodalics) ...
  File "/usr/lib/python3/dist-packages/rocker/core.py", line 230
    print(f"Running docker tag {self.image_id} {image_name}")
                                                           ^
SyntaxError: invalid syntax

dpkg: error processing package python3-rocker (--configure):
 subprocess installed post-installation script returned error exit status 1
Errors were encountered while processing:
 python3-rocker
```

In:

```
$ lsb_release -a
LSB Version:	core-9.20160110ubuntu0.2-amd64:core-9.20160110ubuntu0.2-noarch:printing-9.20160110ubuntu0.2-amd64:printing-9.20160110ubuntu0.2-noarch:security-9.20160110ubuntu0.2-amd64:security-9.20160110ubuntu0.2-noarch
Distributor ID:	Ubuntu
Description:	Ubuntu 16.04.7 LTS
Release:	16.04
Codename:	xenial
```

File: src/rocker/core.py [hunk 1]
```diff
@@ -223,6 +225,10 @@ def build(self, **kwargs):
             arguments['rm'] = True
             arguments['nocache'] = kwargs.get('nocache', False)
             arguments['pull'] = kwargs.get('pull', False)
+            image_name = kwargs.get('image_name', None)
+            if image_name:
+                print(f"Running docker tag {self.image_id} {image_name}")
```
Comment (spd-intermodalics) [inline src/rocker/core.py]:
```suggestion
                print("Running docker tag {} {}".format(self.image_id, image_name))
```

**Review by spd-intermodalics — CHANGES_REQUESTED** [review]
In Ubuntu 16.04, the installation of this package fails for the formatting syntax .

File: src/rocker/core.py (hunk 1 above)
Comment (meyerj) [reply src/rocker/core.py]: :+1:

f-strings were only added in Python 3.6, and 16.04 still has 3.5 only.

**Review by spd-intermodalics — APPROVED** [review]
Thanks! tested

**Comment by meyerj** [comment]
Released and added to http://packages.intermodalics.eu/intermodalics as version `0.2.6-2intermodalics`.
Thanks for testing!

**Pull request merged**